src/eru.Infrastructure/Translation/DependencyInjection.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Globalization;
+using eru.Application.Common.Interfaces;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Localization;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace eru.Infrastructure.Translation
+{
+    public static class DependencyInjection
+    {
+        public static IServiceCollection AddTranslator(this IServiceCollection services)
+        {
+            services.AddLocalization(options =>
+            {
+                options.ResourcesPath = "Resources";
+            });
+            services.Configure<RequestLocalizationOptions>(options =>
+            {
+                var cultures = new[]
+                {
+                    new CultureInfo("en"),
+                    new CultureInfo("pl"),
+                };
+                options.DefaultRequestCulture = new RequestCulture("en");
```

Comment:
```suggestion
                options.DefaultRequestCulture = _configuration["DefaultLanguage"]; 
```

DefaultRequestCulture should be taken from app configuration.

Reply:
moved to #64